content/versions.md

```diff
@@ -89,7 +89,7 @@ announcement](https://blog.chef.io/2019/04/02/chef-software-announces-the-enterp
 <tbody>
 <tr class="odd">
 <td>Chef Automate</td>
-<td>2</td>
+<td>Version 2 Latest build</td>
```

Comment:
```suggestion
<td>Latest</td>
```

Do we need to disambiguate between versions here?

Reply:
was thinking cause of automate 1, but maybe not? i can update it to just latest if yall want

Reply:
Thanks! 1.x is in the EOL table.